depends/patches/boost/mingw32_process.patch

```diff
@@ -0,0 +1,15 @@
+Use Boost.Process with mingw32
```

Comment:
Is this patch from upstream? If so, can you add the upstream commit details / links here? If not, are you planning on upstreaming it? The fact that we still have to patch Process out the box, just to make it work, isn't encouraging. Maybe these fixes can make it into 1.80.0.

Reply:
> If not, are you planning on upstreaming it?

https://github.com/boostorg/process/pull/264